Rephrase definition of single-expression function (#3639)
JetBrains/kotlin-web-site: docs/topics/functions.md

### Single-expression functions

> When a function contains a single `return` expression, the curly braces can be omitted and the body is specified after a `=` symbol:

Reviewer: ```suggestion
When the function body consists of a single expression, the curly braces can be omitted and the body specified after an `=` symbol:
```

Reviewer 2: - []()@_********_
